resources/lrsql/config/prod/default/webserver.edn

```diff
@@ -11,6 +11,13 @@
  :jwt-no-val-issuer         #or [#env LRSQL_JWT_NO_VAL_ISSUER nil]
  :jwt-no-val-role-key       #or [#env LRSQL_JWT_NO_VAL_ROLE_KEY nil]
  :jwt-no-val-role           #or [#env LRSQL_JWT_NO_VAL_ROLE nil]
+ :sec-head-hsts             #or [#env LRSQL_SEC_HEAD_HSTS nil]
+ :sec-head-frame            #or [#env LRSQL_SEC_HEAD_FRAME nil]
+ :sec-head-content-type     #or [#env LRSQL_SEC_HEAD_CONTENT_TYPE nil]
+ :sec-head-xss              #or [#env LRSQL_SEC_HEAD_XSS nil]
+ :sec-head-download         #or [#env LRSQL_SEC_HEAD_DOWNLOAD nil]
+ :sec-head-cross-domain     #or [#env LRSQL_SEC_HEAD_CROSS_DOMAIN nil]
+ :sec-head-content          #or [#env LRSQL_SEC_HEAD_CONTENT nil]
```

Comment:
Just to be clear, NONE of these response headers were set by Pedestal or anything by default before this PR, right?

Reply:
no, none of them were returning. The default return was:
```
Content-Type: application/json;charset=utf-8
Date: Tue, 29 Aug 2023 15:50:25 GMT
Transfer-Encoding: chunked
```